doc/source/whatsnew/v3.0.0.rst

```diff
@@ -473,6 +473,7 @@ Performance improvements
 
 Bug fixes
 ~~~~~~~~~
+- Fixed bug in :meth:`Timestamp.replace` where it would not reflect changes into :meth:`Timestamp.unit`. (:issue:`57749`)
```

Comment:
This should be in the `Datetimelike` section

Reply:
Thanks for the input!